### PR TITLE
Errors

### PR DESCRIPTION
**Pull request by js-kyle**
https://github.com/UniversalMediaServer/api/issues/117
https://github.com/UniversalMediaServer/api/issues/85
https://github.com/UniversalMediaServer/api/issues/122

### CODE REVIEW

**Comment by SubJunk** [comment]
With this I’m going to do a corresponding client change before it merges so they are in sync. One of the downsides of CD

**Comment by js-kyle** [comment]
I'll get it up to date once the other open PR is merged also

**Comment by SubJunk** [comment]
Sweet they're all merged

File: test/e2e/media.spec.ts [hunk 1]
```diff
@@ -181,5 +199,18 @@ describe('Media Metadata endpoints', () => {
       expect(response.body.title).toEqual('Proof of Concept');
       expect(response.body.type).toEqual('episode');
     });
+
+    it('should NOT create a failed lookup document when IMDB api is down', async() => {
```
Comment (SubJunk) [inline test/e2e/media.spec.ts:203]: Very nice work on this, exactly what I was hoping for

**Comment by js-kyle** [comment]
@SubJunk it's up to date now, so ready when the client is

**Pull request merged**